src/bit-systems/object-menu.ts

```diff
@@ -39,10 +54,69 @@ function moveToTarget(world: HubsWorld, menu: EntityID) {
   const targetObj = world.eid2obj.get(ObjectMenu.targetRef[menu])!;
   targetObj.updateMatrices();
 
+  // TODO: position the menu more carefully...
+  //       For example, if a menu object is just placed at a target
+  //       object's position the menu object can be hidden by a large
+  //       target object or the menu object looks too small for a far
+  //       target object.
+  _mat4.copy(targetObj.matrixWorld);
+
+  // Keeps world scale (1, 1, 1) because
+  // a menu object is a child of a target object
+  // and the target object's scale can be changed.
+  // Another option may be making the menu object
+  // a sibling of the target object.
+  _mat4.decompose(_vec3_1, _quat, _vec3_2);
+  _vec3_2.set(1.0, 1.0, 1.0);
+  _mat4.compose(_vec3_1, _quat, _vec3_2);
+
   const menuObj = world.eid2obj.get(menu)!;
+  setMatrixWorld(menuObj, _mat4);
 
-  // TODO: position the menu more carefully...
-  setMatrixWorld(menuObj, targetObj.matrixWorld);
+  // TODO: Remove the dependency with AFRAME
+  const camera = AFRAME.scenes[0].systems["hubs-systems"].cameraSystem.viewingCamera;
+  camera.updateMatrices();
+  menuObj.lookAt(_vec3_1.setFromMatrixPosition(camera.matrixWorld));
+}
+
+// TODO: startRotation/Scaling() and stopRotation/Scaling() are
+//       temporarl implementation that rely on the old systems.
+//       They should be rewritten more elegantly with bitecs.
+function startRotation(world: HubsWorld, targetEid: EntityID) {
+  // TODO: Don't use any
+  // TODO: Remove the dependency with AFRAME
```

Comment:
If you want avoid using any and avoid using the `AFRAME` symbol you can also use:
```javascript
APP.scene!
```

and the add the type to the `aframe.d.ts`:
```javascript
 interface TransformSelectedObjectSystem extends ASystem {
    startTransform(Object3D, EntityID, any): void;
    stopTransform(): void;
  }

  interface AScene extends AElement {
    object3D: Scene;
    renderStarted: boolean;
    renderer: WebGLRenderer;
    tick(time: number, delta: number): void;
    isPlaying: boolean;
    behaviors: {
      tick: AComponent[];
      tock: AComponent[];
    };
    systemNames: Array<keyof AScene["systems"]>;
    systems: {
      "hubs-systems": HubsSystems;
      "personal-space-bubble": PersonalSpaceBubbleSystem;
      "transform-selected-object": TransformSelectedObjectSystem;
      userinput: UserInputSystem;
      /** @deprecated see bit-interaction-system */
      interaction: InteractionSystem;
      nav: NavSystem;
    };
    emit(string, any?): void;
    addState(string): void;
    is(string): boolean;
  }
```

Reply:
There are some other bitecs systems that have AFRAME dependency. I hope we will work on it when we remove the dependency.

Reply:
Agree with Manuel that just adding the type to the aframe.d.ts would be nice. Its fairly minor either way but prevents from making some accidental mistakes leaking the any around.

Reply:
OK, two developers suggested the same thing. And the change is minor. Updated.